c/tskit/core.h

```diff
@@ -301,96 +300,355 @@ not found in the file.
 An unsupported type was provided for a column in the file.
 */
 #define TSK_ERR_BAD_COLUMN_TYPE                                     -105
+/** @} */
 
-
-/* Out of bounds errors */
+/**
+@defgroup OOB_ERROR_GROUP Out of bounds errors.
+@{
+*/
+/**
+A bad value was provided for a ragged column offset, values should
+start at zero and be monotonically increasing.
+*/
 #define TSK_ERR_BAD_OFFSET                                          -200
-#define TSK_ERR_OUT_OF_BOUNDS                                       -201
+/* 201 was the now unused TSK_OUT_OF_BOUNDS */
```

Comment:
We might as well use 201, maybe move TSK_ERR_SEEK_OUT_OF_BOUNDS up here and change its code to 201? That's the least noisy change, and it's vanishingly unlikely anyone is depending on its specific value.